README.md

```diff
@@ -35,6 +35,12 @@ From time to time, one or more assets may be deprecated. In that situation, we w
 * Annotate components with the `to-be-deprecated: <date>` field in their config file setting the date when it will be removed. Add a deprecation notice in this `README.md` file as well.
 * Delete the config file and the associated `data/${name}` folder once the retention period has expired.
 
+### Deprecation notes
+
+### 2023-07-02
+
+Wavefront will no longer be supported.
```

Comment:
I would use a similar approach as we are using for other changelogs:
```suggestion
## Deprecation notes

### 2023-06
Wavefront
```

In this way we don't add an entry per day but per month.

Apart from that, could you please move this section to the end of the README? Just before the License

Reply:
Agree with the list format and the monthly header.